### DISCUSSION TITLE
Welcome to Piranha CMS Discussions!

### DISCUSSION THREAD

**Discussion by tidyui** [PiranhaCMS/piranha.core#1389] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with the members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jensbrak** [comment]
Hello fellow Piranhas!

Seems I got here first, very well then. I am a senior SW developer from Stockholm, Sweden. CS at Uppsala University and then Game Development some years, Enterprise mobility for a decade or so, nowadays consultant. I have used .NET in various forms since about 2005 but do not feel locked to a specific language or paradigm. However, since .NET went open source the world of cool things one can do has exploded, as far as I'm concerned. When I don't program I build a cockpit simulator or LEGO.

As a matter of fact, I found PiranhaCMS when looking for interesting things to learn .NET Core with to prepare for things I need to do with the simulator project later on. Got hooked right right away. All of a sudden I found myself enjoying the benefits of .NET as a multiplatform development tool. I run a personal web using Piranha, hosted on a Raspberry Pi and developed both on a Mac and a PC. Why? Because I can. It's fun. The web page itself has no other purpose than being someting i fiddle with and learn on.

I do have a serious part driving this, the simulator project, which eventually will have to use various API's and hardware to work as intended - but for now It's more of a playground for me. I have worked professionally with older .NET Web versions as well as mobile devices and feel .NET Core is just the right next step - but since work has not provided me with that opportunity I took it privately instead. So here I am.

I find both PiranhaCMS and the small (but growing) community around it to be friendly, helpful, fun and driven by curiosity and generosity. People help and I love to help people if I can so it's a perfect match for me. The downside is that I do this for fun and in the constant need of more time do do everything I want - so my enthusiasm is not always matched by contributions or progress with PiranhaCMS I'm afraid. Still, I keep coming back and do some things here and there to push forward. 

When it comes to Piranha I'd love to dive in deeper to authorization and identity providers. I've played a little with the module concept and comments features and would really enjoy contributing more if opportunity and time allows. And I'd love to see Piranha grow not only with features and users, but with more rich documentation, samples and use cases from other Piranha users here. Eventually, I think I will be able to contribute on that part myself. I am one of these rare wierdos that actually enjoy writing documentation and tutorials (but only if I know what I am talking about - hence I want to do more first).

All the best, 
Jens

**Reply by Achisweden** [reply]
Hello iam 48 years old from Mariestad Sweden. I have a Final Fantasy XI server build on this public project https://gitlab.com/ffxiwings/wings    I am network tech from school, but never work with it. Server been up running little over a month got little help from some friends with install. But now i got stucked when try to fetch latest updates from gitlab maybe there is someone here who can help out looking for a Dev to work with me on server but got stuck atm.  my discord for server 

https://discord.gg/E93nTKzuNQ  

Regards Daniel

**Reply by tidyui** [reply]
Not really sure you've posted this in the right place, but welcome :)

**Reply by jensbrak** [reply]
> Not really sure you've posted this in the right place

I take it that the project web site is built on Piranha, not the project itself.

@Achisweden - if that's correctly assumed there's an issue with that page (https://www.wingsxi.com/) I got a 522 timeout.
Still, the project itself looks rather interesting too :)

**Reply by jensbrak** [reply]
And being in the right place perhaps referes to it being a reply to my presentation? Ah. :)

**Reply by tidyui** [reply]
@jensbrak @Achisweden I might have misread the post, I just read that he wanted help with a server built on top of a completely different open source framework :D

**Comment by tidyui** [comment]
Let me also take the opportunity to introduce myself and the project! I'm the lead developer at byBrick Interface, a digital agency from Västerås Sweden, a Microsoft MVP in Developer Technologies and a Sound Engineer and Music Producer when I get the chance :)

I founded Piranha CMS back in 2011 as the open source options available for .NET at that time was complex, too technical and quite unappealing for the poor content editors. The latest and greatest technology at that time was `ASP.NET MVC 3.0` and `ASP.NET WebPages` powered by the almighty `System.Web`. When Microsoft announced `ASP.NET DNX` I saw the chance to start over from scratch and make all of those changes to the content model that I couldn't do without breaking backwards compatibility and released the first version based on `.NET Core` in late 2017.

I am obsessed with the idea that **simplicity** is always more efficient than complexity and that excellent **user interaction** is the key to success and this mindset has always set the agenda for how we develop new features. As an example, when discussing a new feature we always start with **how** we will present it to the editors in the best way possible. We often start by developing the editorial views in the manager and work on mocked fake data, and only when we get the workflow that we want do we start implementing the actual "technical parts" of the code.

Piranha CMS is my passion project which I have developed and maintained for the last **nine** years and have released a total of **60** versions of. 

From the very beginning my great friend, and **awesome** designer and UX-wizard @NJepop has stood by my side. Without his help the manager interface of Piranha would've have been what it is today and for this I'm ever grateful! Last but not least big thanks to @filipjansson who is my partner in crime when it comes to development and who has to listen to all of my crazy ideas on what we could do :)

**Comment by digityne** [comment]
Hi all! I wasn't sure where else to put this so I thought I'd just say hello and thank you, and let you know my story.

I discovered Piranha just two days ago when looking for a way to add some CMS to a new site I'm building for someone. I've only been a software developer for a few years and a web developer even less (about a year and a half), but I am very impressed with how easy and smooth Piranha has been to use.

I'm using the default MVC project and the architecture has been very intuitive, and thanks to the docs I've already managed to add custom PageTypes to display their business models and add authorised Manager extension modules to edit them, all in fractions of the time it would have taken me to build something from scratch. Couple this with all the CMS functions on the standard pages and I think this would be a winning formula for many people. 

I don't have much to contribute right now other than maybe checking over documents as a native English speaker (my partner is Swedish and I see the same grammatical nuance that she would use) - but I do hope to be able to build more projects and dive deeper into Piranha as time goes on.

Thanks again! Jamie
